Tests/Mapping/Driver/YamlDriverTest.php

```diff
@@ -5,6 +5,9 @@
 
 use Doctrine\Bundle\MongoDBBundle\Mapping\Driver\YamlDriver;
 
+/**
+ * @group legacy
```

Comment:
this seems unrelated, is it some magic group name that allows deprecations to be present?

Reply:
The group tells the Symfony deprecation handler to ignore all deprecation notices that come up during the execution of this test. This can be done to specifically test legacy behaviour without having to define an exception for the deprecation in configuration.